style.css

```diff
@@ -321,3 +321,159 @@ form textarea {
 form button {
   width: 40%;
 }
+
+.desktop-menu {
+  display: none;
+}
+
+/* ---- Desktop Version ---- */
+
+@media screen and (min-width: 768px) {
+  .menu-icon {
```

Comment:
- [OPTIONAL] Kindly make sure the style you added for maintaining your Portfolio responsiveness on the desktop version is not affecting the design of the mobile version. Kindly fix the mobile version design.

   ### YOUR MOBILE DESIGN

     ![screencapture-127-0-0-1-5500-index-html-2023-03-30-19_20_56 (1)](https://user-images.githubusercontent.com/87197412/228915426-724b40ac-2e38-402c-9a14-9a3ef4a855fa.png)